RecoPixelVertexing/PixelLowPtUtilities/test/pcsfVerify.ipynb

```diff
@@ -0,0 +1,414 @@
pL7zXNN\nAAAA827Q8POFJC+tquP3NXTTv98tAwAAOKYd6oYH0/1Gkvcl+UZVXdW1PTDJ7iRPGGJ9AAAA82Kg\n8NNa+3RVnZ3kF6YNc3trkre11rYOt0QAAICjN2jPT7qQ8/rhlgMAADAcg17zk6r60ap6X1VdU1X3\n6dqeWVWPG2qFAAAA82Cg8FNVv5Dkn5Jcm+SsJIu7RRNJXjDcEgEAAI7eoD0/L0jyrNba7ySZmtZ+\neZIHDak2AACAeTNo+PmeJJfN0r4lycnzXBMAAMC8GzT8rE9yziztj0rylXmuCQAAYN4NGn5en+TP\nq+oR3fx9qurpSV6R5K+HWB8AAMC8GPQ5P6+oqlOSfCjJ0iSXJtmR5JWttf85/DIBAACOzkDhp6pO\nSPL7Sf4oyf27HqNrWmtbhl8iAADA0Tts+KmqiSS3JTmvtXZNks+MpjQAAID5c9hrflpru5PckGTJ\naEoCAACYf4Pe8OAPk/xxVd1tyPUAAAAMxUDX/CR5XpKzknyjqr6eZOv0ha21c4dTHgAAwPwYNPy8\nc8h1AAAADNWgt7p+2fBLAQAAGJ5Br/kBAABY0AZ9zs/mJO1gy1trJ89rVQAAAPNs0Gt+fmvG/OIk\n35/kyd2DTwEAAI5pg17z85bZ2qvqiiSPS/IX814ZAADAPDraa34uTfLEeaoFAABgaI42/DwlyS3z\nVAsAAMDQDHrDg6tn3PCgktw9yV2S/MbwygMAAJgfR/qQ0z1JNiRZ01r74hDqAgAAmFcecgoAAPTC\nQNf8VNWKqloxbf6BVfXyqnrqUKsDAACYJ4Pe8OCf9t3VraruluTjSf5rkr+pqucOt0QAAICjN2j4\nOTfJ5d30Tyf5j9baA5I8LcmvD7E+AACAeTFo+Dk+yZZu+r8keW83fUWS+wypNgAAgHkzaPi5NslP\nVdV9kvxIkku69rsnuXWI9QEAAMyLQcPPy5L8SZKvJrm8tfaprv3xST47xPoAAADmxaC3un5XVZ2R\n5J5Jrpy26MNJ/nnQg1XVcd3NEpZ0P+9prb34iCoHAACYg0EfcprW2s1Jbp7R9qmDbzHrPnZU1WNa\na9uqaiLJJ6rqEa21T8xlPwAAAHM1cPipqnO6O72d0fXa7Ndae8ag+2mtbesmj+uG3W2cS8EAAABH\nYqDwU1VP6Ia3fTbJ+Uk+neS7uwDzb3M5YFUtSrK22/5vWmvXHHH1AAAAAxq05+cPkrystfY/qmpz\nkl9Ksj7J3ye5bC4HbK3tSfL9VXVykkuq6tGttY/NXG/16tX7p1etWpVVq1bN5TBwVN7whn/J+9//\nqXz965syNVX727/97duydeu2tDaRqamptLa3fffu3fun99mzp92h7ViyZ8/OJIuzZ8+uaf8VbMrX\nv/7hOexld5KJIVV4LKr9f1YlVXvnZ04vWlSH2MccjlbJxMSRv79Vu7Jly/HzUstCULU7mzYtHXcZ\nWbRoKps2LZt12cREZevWozsnk5MtO3acclT7mM2SJZWqux50+dKlE/nqV0+f9+MeiWXLFmXDhvnd\n5/LlSwZYCxinNWvWZM2aNUe1j2oDfDqrqi1Jzm2tXVdV30nyqNba56vqgUne31o744gOXvWSJNta\na6+a0d4GqQuG5eUvf3s+97lk27bzcsIJK/e3X3PNjZma2prJyZXZvHlrJif3fsDZtu32TEwc+IFm\n+/btmZgY/wexPtu9+/YsXXrwD5q7d6/LkiXbctJJ5x90nV27vpXly0/L1q3/J/e//8Oybdu3cuaZ\np2XLlutzzjlnJUk2bVqb887bu4+NG9flggtWHnR/c7Fhw9pceOHBawOAPquqtNbm9I3joLe63pxk\n36e4m5Lct5ueTHLqHAq8W1Wd0k0fn+SHk3xuLgUDAAAciUGHvX0qySOTXJPk/UleVVXnJfmvcxz2\ndo8kb6m940MWJfn71tpHjrB2AACAgQ0afn43yYnd9OokJyV5cpIvd8sG0lq7OsmDj6xUAACAIzfo\nQ06vmza9LclvDLUqAACAeTboNT+pqqVV9dNV9cKqWt61fXdV3WWoFQIAAMyDQZ/zc98kH+6Gvi1P\n8o4kt3Y9QMuTPHP4pQIAABy5QXt+XpPkkiR3T3L7tPb3JnnMkGoDAACYN4Pe8OCCJA9vre3e9yC/\nzo1J7jmc0gAAAObPwNf8JFk8S9sZSW6bx3oAAACGYtDwc8mMW1q3qjo5ycu65/4AAAAc0+bynJ9L\nq+pLSZYm+cck901yc5KfHXKNAAAAR23Q5/ysr6oHJXlq95DSRUlen+StrbXbB9gFAADAWA3a85Mu\n5Pxt9wMAALCgDBx+quq0JI9MctrMa4Vaa381lOoAAADmyaAPOX1q1+OzKMnGJG3a4pZE+AEAAI5p\ng/b8/HGSP03yB621qSHXBAAAMO8GvdX1KUneLPgAAAAL1aDh5x+SPGHItQAAAAzNoMPefjvJe6vq\ncUmuTrJr+sLW2h8MpzwAAID5MWj4+bUkP5zklu7hpjNveCD8AAAAx7RBw89Lkjy3tfbqIdcDAAAw\nFINe8zOR5L1DrgUAAGBoBg0/FyX5hSHXAgAAMDSDDns7Ickzq+rxSa6a5YYHzx5OeQAAAPNj0PCz\nMslnu+n7zVjWZlkfAADgmDJQ+GmtPWb4pQAAAAzPoNf8AAAALGjCDwAA0AvCDwAA0AvCDwAA0AvC\nDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA\n0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvC\nDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AvCDwAA0AsjDT9V\nde+q+mhVfaGqrq6qZ4/y+AAAQH9
```

Comment:
the png is generated by jupyter itself. I can try to clear the output. it defeats the self documenting purpose though